src/workers/parser/frameworks.js

```diff
@@ -4,7 +4,13 @@
 
 import getSymbols from "./getSymbols";
 
-export function isReactComponent(source) {
+export function whichFramework(source) {
```

Comment:
i think i would prefer `getFramework`. We use `get` and `fetch` for most of our query functions

Reply:
Ok, will change it to match.